frontend/src/app/supportAdmin/DeviceType/ManageDevicesForm.tsx

```diff
@@ -133,13 +162,14 @@ const ManageDevicesForm: React.FC<Props> = ({
                     required
                   />
                 </div>
-              </div>
-              <div className="grid-row grid-gap">
                 <div className="tablet:grid-col">
                   <TextInput
-                    label="Model"
-                    name="model"
-                    value={selectedDevice?.model}
+                    type={"number"}
+                    label="Test Length"
```

Comment:
I wonder if this label should be "Test Length (minutes)" to match what Tim asked for [here](https://github.com/CDCgov/prime-simplereport/issues/3414#issuecomment-1064538424) on the create device type form